packages/orchestration/src/exos/chain-hub.js

```
@@ -194,8 +239,10 @@ const ChainHubI = M.interface('ChainHub', {
  * @param {Zone} zone
  * @param {Remote<NameHub>} agoricNames
  * @param {VowTools} vowTools
+ * @param {(...args: unknown[]) => void} [log] TODO - only for tests - should we
+ *   update withOrchestration so we can observe these?
```

I wouldn't say "only for tests"; it's for all the things that explicit authority is good for.

---

Removed since we now throw instead of log